Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Remove non-working echo entries from documentation makefile.

diff -Naurp crtools.orig/Documentation/Makefile crtools/Documentation/Makefile
--- crtools.orig/Documentation/Makefile
+++ crtools/Documentation/Makefile
@@ -11,17 +11,14 @@ MAN8DIR		:= $(MANDIR)/man8
 all: $(MANS)
 
 %.8: %.txt
-	$(E) "  GEN     " $@
 	$(Q) $(ASCIIDOC) -b docbook -d manpage -o $(patsubst %.8,%.xml,$@) $<
 	$(Q) $(XMLTO) man --skip-validation $(patsubst %.8,%.xml,$@) 2>/dev/null
 
 clean:
-	$(E) "  CLEAN   "
 	$(Q) rm -f ./*.xml
 	$(Q) rm -f ./*.8
 
 install: $(MANS)
-	$(E) "  INSTALL " $(MANS)
 	$(Q) mkdir -p $(DESTDIR)$(MAN8DIR)
 	$(Q) install $(MANS) $(DESTDIR)$(MAN8DIR)
 
