Description: Fix FTBFS with clang
Bug-Debian: http://bugs.debian.org/753845
Forwarded: no
Author: Arthur Marble <arthur@info9.net>
Reviewed-by: Salvatore Bonaccorso <carnil@debian.org>
Last-Update: 2014-07-30

--- a/arch/x86/crtools.c
+++ b/arch/x86/crtools.c
@@ -33,12 +33,6 @@ const char code_syscall[] = {
 
 const int code_syscall_size = round_up(sizeof(code_syscall), sizeof(long));
 
-static inline void __check_code_syscall(void)
-{
-	BUILD_BUG_ON(sizeof(code_syscall) != BUILTIN_SYSCALL_SIZE);
-	BUILD_BUG_ON(!is_log2(sizeof(code_syscall)));
-}
-
 void parasite_setup_regs(unsigned long new_ip, void *stack, user_regs_struct_t *regs)
 {
 	regs->ip = new_ip;
--- a/cr-show.c
+++ b/cr-show.c
@@ -205,18 +205,6 @@ static void show_collect_pstree(struct c
 			collect ? pstree_handler : NULL, "*:%d");
 }
 
-static inline char *task_state_str(int state)
-{
-	switch (state) {
-	case TASK_ALIVE:
-		return "running/sleeping";
-	case TASK_DEAD:
-		return "zombie";
-	default:
-		return "UNKNOWN";
-	}
-}
-
 static void show_core_regs(UserX86RegsEntry *regs)
 {
 #define pr_regs4(s, n1, n2, n3, n4)	\
